man/waybar-mpris.5.scd

```diff
@@ -71,12 +71,27 @@ The *mpris* module displays currently playing media via libplayerctl.
 	something less than or equal to this value, so the title will always be ++
 	displayed.
 
-*dynamic-priority*: ++
+*dynamic-order*: ++
 	typeof: []string ++
-	default: ["title", "length", "position", "artist", "album"] ++
-	Priority of the tags when truncating the Dynamic tag (absence in this
+	default: ["title", "artist", "album", "position", "length"] ++
+	Order of the tags shown by Dynamic tag. The position and length tags ++
+	will always be combined in the format [{position}/{length}]. The order ++
+	of these tags in relation to other tags will be determined based on the ++
+	declaration of the first among the two tags. Absence in this list means ++
+	force exclusion.
+
+*dynamic-truncation-priority*: ++
```

Comment:
at the risk of bikeshedding here, I don't really like the name `dynamic-truncation-priority` because it implies that high priority = gets truncated first, while it's precisely the opposite here (high priority = gets truncated last)

Reply:
What I understood was that the first on the list will be truncated before. Do you have another name suggestion?

Reply:
maybe dynamic-truncation-order?

Reply:
maybe dynamic-importance?